xcp_d/cli/run.py

```diff
@@ -23,7 +23,7 @@ def main():
         write_dataset_description,
     )
 
-    parse_args()
+    parse_args(args=sys.argv[1:])
```

Comment:
Need to pass the arguments in directly so I can patch them in in the integration tests.